modules/tinymce/src/core/main/ts/focus/FocusController.ts

```diff
@@ -59,11 +59,11 @@ const registerEvents = (editorManager: EditorManager, e: { editor: Editor }) =>
 
   SelectionRestore.register(editor);
 
-  const toggleContainerFocus = (editor: Editor) => {
+  const toggleContentAreaOnFocus = (editor: Editor, fn: (element: SugarElement<Element>, clazz: string) => void) => {
```

Comment:
I'm not sure exactly how, but I feel like `Class.toggler()` will make this easier. Or maybe just using the `Toggler` module directly, since there's extra processing required each time.

Reply:
I don't think it will ever fire focusin twice in a row. Also the the `classList.add` will only add the class if it's not there a toggler would first read the value to check if it's there then toggle it on so that would be two dom operations so likely slower. I think the code is pretty easy to understand as it is.

Reply:
I would choose the current code because it is slightly faster

Reply:
I just don't like the style of passing `Class.add` and `Class.remove` as first-class arguments